### PR TITLE
Make it possible to set hosts in different ways.

### PR DESCRIPTION
**Pull request by ruflin**
* Add tests for the different hosts options.
* Add default config file for documentation purpose.
* Apply goimports*

### CODE REVIEW

File: outputs/elasticsearch/output.go [hunk 1]
```diff
@@ -282,3 +290,56 @@ func (out *elasticsearchOutput) BulkPublish(
...
+	port := ""
+
+	// If url doesn't have a scheme, host is written into path
+	if urlStruct.Host == "" {
+		urlStruct.Host = urlStruct.Path
+		urlStruct.Path = ""
+	}
+
+	// Check for port
+	i := strings.Index(urlStruct.Host, ":")
```
Comment (urso) [inline outputs/elasticsearch/output.go]: what if host is IPv6 address?

File: outputs/elasticsearch/output_test.go [hunk 2]
```diff
@@ -332,3 +333,26 @@ func TestEnableTTL(t *testing.T) {
...
+	inputOutput := map[string]string{
+		"":                         "http://localhost:9200",
+		"http://localhost":         "http://localhost:9200",
+		"http://localhost/":        "http://localhost:9200/",
+		"http://192.168.1.1:9200":  "http://192.168.1.1:9200",
+		"https://192.168.1.1:9200": "https://192.168.1.1:9200",
+		"http://192.168.1.1":       "http://192.168.1.1:9200",
+		"http://192.168.1.1/hello": "http://192.168.1.1:9200/hello",
+		"192.168.1.1":              "http://192.168.1.1:9200",
+		"192.168.1.1:9200":         "http://192.168.1.1:9200",
```
Comment (urso) [inline outputs/elasticsearch/output_test.go]: add tests for IPv6 addresses

**Comment by ruflin** [comment]
@urso IPv6 support added

File: outputs/elasticsearch/output.go [hunk 3]
```diff
@@ -282,3 +290,70 @@ func (out *elasticsearchOutput) BulkPublish(
...
+	// If url doesn't have a scheme, host is written into path
+	if urlStruct.Host == "" {
+		urlStruct.Host = urlStruct.Path
+		urlStruct.Path = ""
+	}
+
+	// Check for port
+	i := strings.Count(urlStruct.Host, ":")
+
+	ipv6 := false
```
Comment (urso) [inline outputs/elasticsearch/output.go]: maybe safer to move ipv6 test after splithostport on extracted host value

Comment (ruflin) [inline outputs/elasticsearch/output.go]: Done

**Pull request merged**